Unused variable warnings when building

When building this package (ROS 2), I see these warnings.

```
/home/runner/work/moveit2/moveit2/.work/upstream_ws/src/geometric_shapes/src/aabb.cpp:43:26: warning: unused variable 't' [-Wunused-variable]
     43 |   const Eigen::Vector3d& t = transform.translation();
        |                          ^
  /home/runner/work/moveit2/moveit2/.work/upstream_ws/src/geometric_shapes/src/aabb.cpp:45:10: warning: unused variable 'x_range' [-Wunused-variable]
     45 |   double x_range = 0.5 * (fabs(r(0, 0) * box[0]) + fabs(r(0, 1) * box[1]) + fabs(r(0, 2) * box[2]));
        |          ^~~~~~~
  /home/runner/work/moveit2/moveit2/.work/upstream_ws/src/geometric_shapes/src/aabb.cpp:46:10: warning: unused variable 'y_range' [-Wunused-variable]
     46 |   double y_range = 0.5 * (fabs(r(1, 0) * box[0]) + fabs(r(1, 1) * box[1]) + fabs(r(1, 2) * box[2]));
        |          ^~~~~~~
  /home/runner/work/moveit2/moveit2/.work/upstream_ws/src/geometric_shapes/src/aabb.cpp:47:10: warning: unused variable 'z_range' [-Wunused-variable]
     47 |   double z_range = 0.5 * (fabs(r(2, 0) * box[0]) + fabs(r(2, 1) * box[1]) + fabs(r(2, 2) * box[2]));
        |          ^~~~~~~
```

Leaving as a reminder to fix this later, but if someone gets to it first that's cool.

--- a/src/aabb.cpp
+++ b/src/aabb.cpp
@@ -34,18 +34,6 @@
 
 void bodies::AABB::extendWithTransformedBox(const Eigen::Isometry3d& transform, const Eigen::Vector3d& box)
 {
-  // Method adapted from FCL src/shape/geometric_shapes_utility.cpp#computeBV<AABB, Box>(...) (BSD-licensed code):
-  // https://github.com/flexible-collision-library/fcl/blob/fcl-0.4/src/shape/geometric_shapes_utility.cpp#L292
-  // We don't call their code because it would need creating temporary objects, and their method is in floats in FCL 0.5
-  //
-  // Here's a nice explanation why it works: https://zeuxcg.org/2010/10/17/aabb-from-obb-with-component-wise-abs/
-  const Eigen::Matrix3d& r = transform.rotation();
-  const Eigen::Vector3d& t = transform.translation();
-
-  double x_range = 0.5 * (fabs(r(0, 0) * box[0]) + fabs(r(0, 1) * box[1]) + fabs(r(0, 2) * box[2]));
-  double y_range = 0.5 * (fabs(r(1, 0) * box[0]) + fabs(r(1, 1) * box[1]) + fabs(r(1, 2) * box[2]));
-  double z_range = 0.5 * (fabs(r(2, 0) * box[0]) + fabs(r(2, 1) * box[1]) + fabs(r(2, 2) * box[2]));
-
   fcl::AABBd aabb;
   fcl::computeBV(fcl::Boxd(box), transform, aabb);
   extend(aabb.min_);

